plugins/chunter-resources/src/components/DirectMessageButton.svelte

```diff
@@ -20,8 +20,14 @@
   import chunter from '../plugin'
   import { createDirect } from '../utils'
   import { openChannelInSidebar } from '../navigation'
+  import { Asset } from '@hcengineering/platform'
 
   export let employee: Employee
+  export let width: string | undefined = undefined
+  export let kind: 'primary' | 'secondary' | 'tertiary' | 'negative' = 'secondary'
+  export let isButtonIcon: boolean = false
```

Comment:
Can remove it and check only `type` ?

Reply:
Sure, fixed

Reply:
https://github.com/hcengineering/platform/pull/8504/files#diff-b822690f53209901fed46e900768abb35d7c74708c603cc3c500ec039f544247R74

Can we remove it everywhere?

Reply:
@ArtyomSavchenko looks like you missed this comment

Reply:
Thanks, fixed